### PR TITLE
[DOCS] Update CODEOWNERS, Some improves of modelzoo.md

### PR DESCRIPTION
**Pull request by AlexanderDokuchaev**
### Changes

- Fix names of teams in CODEOWNERS
    - But openvino-admins and openvino-configuration-mgmt looks like private groups as result 
     ```
    Unknown owner on line 3: make sure the team @openvinotoolkit/openvino-admins exists, is publicly visible, and has write access to the repository
    ```

- Little improves of modelzoo.md

### CODE REVIEW

**Review by alexsu52 — APPROVED** [review]
LGTM

**Review by nikita-savelyevv — APPROVED** [review]

File: CODEOWNERS [hunk 1]
```diff
@@ -1,7 +1,7 @@
-*  @openvinotoolkit/nncf_pytorch-maintainers
+*  @openvinotoolkit/nncf-maintainers
 
 CODEOWNERS  @openvinotoolkit/openvino-admins
```
Comment (vshampor) [inline CODEOWNERS:3]: Should probably point to `nncf-admins`

File: CODEOWNERS [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 
 # Control 3d party dependencies
-requirements.txt  @openvino-configuration-mgmt
-**/setup.py  @openvino-configuration-mgmt
+requirements.txt  @openvinotoolkit/openvino-configuration-mgmt
+**/setup.py  @openvinotoolkit/openvino-configuration-mgmt
```
Comment (vshampor) [inline CODEOWNERS:7]: Remove this altogether. No one in these teams seemed to care before and I don't see why they would be caring now; at least there is no defined process for it.

**Review by vshampor — CHANGES_REQUESTED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (AlexanderDokuchaev) [reply CODEOWNERS]: Yes, looks more actual 👍

File: CODEOWNERS (hunk 2 above)
Comment (AlexKoff88) [reply CODEOWNERS]: I am ok

Comment (alexsu52) [reply CODEOWNERS]: I'm ok.

**Review by vshampor — APPROVED** [review]

**Pull request merged**